### DISCUSSION TITLE
Improve compatibility for package manger and portable

### DISCUSSION THREAD

**Discussion by lyaaz** [ppy/osu#14790] [Ideas]
1. To use package manager such as [scop](https://github.com/lukesampson/scoop) and [choco](https://chocolatey.org/) to manager osulazer, auto update should be disabled to avoid conflict.
2. allow use args when startup to configure data path, for example use `osu.exe -path D:\osu-lazer` to specify the storage folder.

**Comment by peppy** [comment]
Disabling auto-update is out of scope for now. You can set a custom data path using `storage.ini`.

**Reply by lyaaz** [reply]
> You can set a custom data path using `storage.ini`.

yes, but it's not that convenient to use if I install osu in a usb stick. With args I can use a script to start osu without changing anything.
